Readme.md

```diff
@@ -72,7 +75,8 @@ Here's an overview of the operating systems supported in v86:
   - [SkiffOS](https://github.com/skiffos/SkiffOS/tree/master/configs/browser/v86) (based on Buildroot) can cross-compile a custom image.
   - Archlinux works. See [archlinux.md](docs/archlinux.md) for building an image.
   - Debian works. An image can be built from a Dockerfile, see [tools/docker/debian/](tools/docker/debian/).
-  - Ubuntu up to 16.04 works.
+  - Ubuntu up to 16.04 LTS works.
+  - Lubuntu/Ubuntu Budgie/Kubuntu up to 18.04 LTS
```

Comment:
Does regular ubuntu 18.04 work? I'd compress this to "Ubuntu up to 18.04 LTS works." and remove the unnecessary mentions of all the ubuntu derivatives.

Reply:
In official Ubuntu, the latest x86 version became [16.04](https://releases.ubuntu.com/16.04/), but in some distributions 32-bit versions remained until 18.04 (for example, [Kubuntu](https://cdimage.ubuntu.com/kubuntu/releases/18.04.5/release/))

Reply:
in the case of Lubuntu, 18.10 (but it's not supported anymore)

Reply:
Same with Ubuntu Server 17.10 (see http://old-releases.ubuntu.com/releases/17.10/)